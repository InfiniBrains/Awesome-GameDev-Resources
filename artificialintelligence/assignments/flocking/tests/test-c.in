0.000 0.000 0.000 2.000 0.000 0.000 2.000 2
0.000 0.500 3.000 0.000
0.000 -0.500 -2.000 0.000
0.125
