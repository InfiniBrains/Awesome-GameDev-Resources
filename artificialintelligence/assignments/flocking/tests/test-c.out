0.391 0.500 3.125 0.000
-0.234 -0.500 -1.875 0.000
